plugins/test/chromatic_correction_test.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright © 2024 Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import unittest
+
+import numpy
+from odemis import model
+
+from odemis.dataio.tiff import read_data
+from odemis.util.spot import MaximaFind
+from odemis.util.transform import AffineTransform, alt_transformation_matrix_to_implicit
+from plugins.chromatic_correction import (ACTUAL_BEAD_DIAMETER, SCALE,
+                                          BEAD_QUANTITY, get_chromatic_correction_dict)
+
+
+class TestChromaticAberrationCorrection(unittest.TestCase):
+    """
+    Test the Chromatic Aberration Correction functions used in the plugin
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.das = []
+        for input_file_path in ['FOV1_green.tif', 'FOV1_blue.tif']:
+            cls.das.append(read_data(input_file_path))
+
+    def test_get_chromatic_correction_dict(self):
+        max_dist = ACTUAL_BEAD_DIAMETER * SCALE  # size of bead in the image
+        ref = self.das[0][0]
+        im = self.das[1][0]
+        im.metadata.update({model.MD_PIXEL_SIZE: (0.07738095984842844 * 1e-6, 0.07738095984842844 * 1e-6)})
+
+        # convert m to pixel units
+        max_dist_pxl = int(max_dist / im.metadata[model.MD_PIXEL_SIZE][0])
+
+        # Find transformation values per channel
+        points_ref = MaximaFind(ref, BEAD_QUANTITY, max_dist_pxl)
+        points_im = MaximaFind(im, BEAD_QUANTITY, max_dist_pxl)
+        _ = get_chromatic_correction_dict(im, points_ref, points_im)
```

Comment:
As second test could be to pass points_ref & points_ref as argument. The computed metadata should be the identity.